src/biokbase/narrative/jobs/appmanager.py

```diff
@@ -226,7 +202,7 @@ def _run_app_internal(self, app_id, params, tag, version, cell_id, run_id, **kwa
         self.spec_manager.check_app(app_id, tag, raise_exception=True)
 
         if version is not None and tag != "release":
-            if re.match(version, '\d+\.\d+\.\d+') is not None:
+            if re.match(r'\d+\.\d+\.\d+', version) is not None:
                 raise ValueError("Semantic versions only apply to released app modules. You can use a Git commit hash instead to specify a version.")
```

Comment:
I switched to 100 char lines in Java code recently and maintain this rule for all new code. For old code it's a matter of time of course.